declarations/Grindr.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
Updated declaration in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/e6a4a193374eb5929083ca5bed70913b91879ffe